code/WorkInProgress/rewardsLocker.dm

```diff
@@ -1134,6 +1134,77 @@
 		else
 			boutput(activator, "<span class='alert'>You need to be an AI to use this, you goof!</span>")
 
+
+////////AI CORE FRAMES///////
+
+/datum/achievementReward/ai_bee
```

Comment:
I think with this, it's time. Please move these AI ones to a separate file / move rewardsLocker.dm to `modules/medals` or something